src/irmin-graphql/server.ml

```diff
@@ -321,7 +331,7 @@ struct
                   Store.Tree.find tree path >|= Result.ok);
               io_field "get_contents"
                 ~args:Arg.[ arg "path" ~typ:(non_null Input.path) ]
-                ~typ:Lazy.(force contents)
```

Comment:
This is great! Removing all the lazy values makes this file much easier to understand.

Reply:
`Schema.add_type` mutates the GraphQL schema, so you shouldn't be using it inside resolvers. If you introspect the schema before and after running this resolver, you'll observe different results. So unfortunately you're not getting rid of the lazy values just yet. 🙁

I have some ideas for handling this better in the `graphql` package though..

Reply:
Ah thanks for taking a look! @patricoferris I can update this PR to move `Schema.add_type` outside of the resolver.

Reply:
I just pushed a commit to make `store_schema` lazy, which allows for `Schema.add_type` to be called at the global scope instead of in the resolver: https://github.com/mirage/irmin/pull/1843/commits/8fbb0d2a4cbf2d03e09c0d825164bbb79da63b0f

Reply:
Thanks @andreas for looking over the PR and @zshipko for the fixes :))